docs/concepts/assets/accounts.md

```diff
@@ -24,6 +24,35 @@ Any assets that are in a general account can be withdrawn or transferred.
 **[REST](../../api/rest/data-v2/trading-data-service-get-party.api.mdx)**: Use the API to check your public key for the accounts your assets are allocated to.
 :::
 
+## Vault
+A *vault* holds assets that have been contributed by one or more parties to be used by another party for trading. The assets in a vault can only be used to enter and exit positions. Vaults have general and margin accounts associated with them.
+
+Contributors to a vault have a share of the value in the vault, based on how much their deposited amount contributed to its original value.
+
+The vault account manager - the party that created the account - can charge fees on account withdrawals if they choose to.
```

Comment:
is account manager how we want to call the party who created the vault?

Reply:
> can charge fees on account withdrawals if they choose to.
not on withdrawals, fees are on performance and on management not on withdrawals.

Reply:
you may also want to add that the vault owner (account manager) can transfer ownership of the vault.